admin/server/auth/claims.go

```diff
@@ -97,26 +124,25 @@ func (c *authTokenClaims) CanOrganization(ctx context.Context, orgID string, p O
 	t := c.token.Token().Type
 	switch t {
 	case authtoken.TypeUser:
-		role, err := c.composeOrgPermissions(ctx, orgID)
+		permissions, err := c.OrganizationPermissions(ctx, orgID)
 		if err != nil {
-			// TODO: log error
-			return false
+			panic(fmt.Errorf("failed to get organization permissions: %w", err))
 		}
```

Comment:
If you do the `panic` inside `c.OrganizationPermissions` instead of returning errors, you would also get nicer handling in the server handlers, like `Permissions: claims.OrganizationPermissions(ctx, org.ID)`